docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -41,14 +41,16 @@ To isolate a host from a case alert:
 
 To isolate a host from an endpoint:
 
-. Go to *Manage -> Endpoints*, then select the appropriate endpoint in the *Endpoint* column. The endpoint details flyout opens.
-. Click *Take action -> Isolate host*.
+. Go to *Manage -> Endpoints*, then either:
+    * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Isolate host* in the endpoint details flyout.
+    * Click the *Actions* icon (*...*) for the appropriate endpoint, then select *Isolate host*.
 . Enter a comment describing why you’re isolating the host (optional).
 . Click *Confirm*.
 
-After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*.
+After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any user currently on the host receives a notification that the computer has been isolated from the network:
```

Comment:
Just a suggestion here. Debating b/t "any active end user" and "an active end user." 
```suggestion
After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any active end user on the host receives a notification that the computer has been isolated from the network:
```

Reply:
+ "active end user". I think "An active end user" feels a bit nonspecific, so that's why I originally went with "Any...."